homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -251,6 +251,13 @@ async def async_step_auth(
                 url = await self.flow_impl.async_generate_authorize_url(self.flow_id)
         except asyncio.TimeoutError:
             return self.async_abort(reason="authorize_url_timeout")
+        except NoURLAvailableError:
+            return self.async_abort(
+                reason="no_url_available",
+                description_placeholders={
+                    "docs_url": "https://www.home-assistant.io/docs/configuration/basic/"
```

Comment:
We should add a dedicated help page that explains what's going on. It can then link to this url . We can add a place holder for now to the main docs.